File: is28/vyacheslavleva28/lab12/prog3.cpp
#include <math.h>
#include <iostream>

using namespace std;

int main()
{
	int d;
	
	cin >> d;
	
	if ( d % 7 == 0 )  
	{ 
		cout << "пн" << endl;
	} else if ( d % 7 == 1 )
	{
		cout << "вт" << endl;
	} else if ( d % 7 == 2 )
	{
		cout << "ср" << endl;
	} else if ( d % 7 == 3 )
	{ 
		cout << "чт" << endl;
	} else if ( d % 7 == 4 )
	{
		cout << "пт" << endl;
	} else if ( d % 7 == 5 )
	{
		cout << "сб" << endl;
	} else 
	{
		cout << "вс" << endl;
	}
	
	return 0;
}

File: is28/sharutin28/lab3/prog3.py
import math
d = int(input())
if 365 % d == 0:
	print('1 пн')
elif 365 % d == 1:
	print('2 вт')
elif 365 % d == 2:
	print('3 ср')
elif 365 % d == 3:
	print('4 чт')
elif 365 % d == 4:
	print('5 пт')
elif 365 % d == 5:
	print('6 сб')
else :
	print('7 вс')
	